runtime/runtime/src/metrics.rs

```diff
@@ -313,6 +338,40 @@ fn buckets_for_compute() -> Option<Vec<f64>> {
     ])
 }
 
+fn buckets_for_storage_proof_size() -> Vec<f64> {
```

Comment:
nit: I would consider `exponential_buckets` here and below to minimise number of code lines

Reply:
Done

Reply:
I would recommend decreasing the range and step size. Currently most "interesting" values are in the range 1MB - 16MB, so having more granular step would be nice there. Anything below 100KB is pretty much the same and can be classified as "very small", the same for anything above 16MB.
So I would start with 100KB and have multiplier of something like 1.2-1.5, up to you.

Reply:
> I would recommend decreasing the range and step size. Currently most "interesting" values are in the range 1MB - 16MB, so having more granular step would be nice there. Anything below 100KB is pretty much the same and can be classified as "very small", the same for anything above 16MB. So I would start with 100KB and have multiplier of something like 1.2-1.5, up to you.

Ok, I can remove the smaller buckets in the chunk-level metrics, but I'll keep them in the receipt-level ones, receipts should consume much less than 1MB (although you never know x.x)